Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+
+public enum TransformType: Int {
+    case resetTransforms
+    case transform
+}
+
+public protocol TransformDelegate : AnyObject {
+    
+    var undoButton: UIBarButtonItem! { get set }
+    var redoButton: UIBarButtonItem! { get set }
+    var resetButton: UIBarButtonItem! { get set }
+    
+    func undoManager() -> UndoManager
+    func isUndoEnabled() -> Bool
+    func isRedoEnabled() -> Bool
+    func isUndoing() -> Bool
+    func isRedoing() -> Bool
+    func undo()
+    func redo()
+    func updateCropState(_ cropState: Any)
+    func enableResetButton(_ enable: Bool)
```

Comment:
Maybe we can rename it to `enableReset` to remove the UI concept from the protocol and make real delegate to decide either enable a reset button or other UI controls, but protocol does not need to limit it as a button.

Reply:
renamed to "enableReset"